### PR TITLE
Bug 1779858: Fixed required fields display issue for create operand form

### PR DESCRIPTION
**Pull request by yzhao583**
Fixes: https://issues.redhat.com/browse/CONSOLE-1927

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx [hunk 1]
```diff
@@ -253,8 +253,8 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = ({
           }
           const schemaPath = field.path.split('.').join('.properties.');
           const required = (_.get(
-            openAPI,
-            _.dropRight(['properties', 'spec', ...field.path.split('.')])
+            openAPI.properties,
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx]: Will `openAPI` ever be null? We might want

```suggestion
            openAPI?.properties,
```

Comment (yzhao583) [reply frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx]: @spadgett Thanks Sam! It is a good suggestion.

**Pull request merged**